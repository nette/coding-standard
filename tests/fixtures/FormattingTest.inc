<?php // {"optimizedFunctionsOnly": false}

namespace App;

use stdClass;


class MyClass
{
    public function run()
    {
        \is_string('foo');
    }
}
