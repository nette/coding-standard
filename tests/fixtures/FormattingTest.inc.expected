<?php // {"optimizedFunctionsOnly": false}

namespace App;

use stdClass;
use function is_string;


class MyClass
{
    public function run()
    {
        is_string('foo');
    }
}
